src/client/components/Story/StoryFull.js

```diff
@@ -263,6 +264,43 @@ class StoryFull extends React.Component {
       );
     }
 
+    let postedFrom;
```

Comment:
This logic is exactly the same in `Story` as well. We can create a component to do just that, so we don't have duplicate code.